src/docs/development/packages-and-plugins/developing-packages.md

```diff
@@ -342,6 +347,19 @@ This is done using a [platform channel][],
 or through the interfaces defined in a platform
 interface package.
 
+### Add support to new platforms in an existing plugin project
+
+To support new platforms in an existing plugin project, run `flutter create` with
+the `--template=plugin` flag again in the project directory.
+For example, to add web support in an existing plugin, run:
+
+```
+$ flutter create --template=plugin --platforms=web .
+```
+
+If successful, the terminal might show a message about updating the `pubspec.yaml`.
+Follow the instruction there to update the current `pubspec.yaml` file.
+
```

Comment:
In case they didn't read the Step 1 note, maybe:
```
{{site.alert.important}}
  The `--platforms ` flag is available in Flutter 1.20.0.
{{site.alert.end}}
```

Reply:
done